app/Helpers/LogHelper.php

```diff
@@ -1862,21 +1868,27 @@ public static function processEmployeeLog(EmployeeLog $log): string
                 break;
 
             case 'meeting_decision_updated':
-                $sentence = trans('account.employee_log_log_meeting_decision_updated', [
+                $sentence = trans('account.employee_log_meeting_decision_updated', [
                     'group_id' => $log->object->{'group_id'},
                     'group_name' => $log->object->{'group_name'},
                     'meeting_id' => $log->object->{'meeting_id'},
                 ]);
                 break;
 
             case 'add_guest_to_meeting':
-                $sentence = trans('account.employee_log_log_add_guest_to_meeting', [
+                $sentence = trans('account.employee_log_add_guest_to_meeting', [
```

Comment:
This is just a quick fix for the audit log, not relevant to this PR but I dont want to create a separate PR just for that.